src/gen3-ui-component/components/filters/FilterGroup/index.jsx

```
@@ -145,8 +144,43 @@ function FilterGroup({
     return capitalizeFirstLetter(newFilterName);
   }
 
-  const filterToRelation = filterConfig.filterDependencyConfig.filterToRelation;
-  const relations = filterConfig.filterDependencyConfig.relations;
+  // default pulled from pcdc.json
+  const FALLBACK_FILTER_DEPENDENCY_CONFIG = {
+    filterToRelation: {
+      'molecular_analysis.molecular_abnormality': 'molecular_abnormality',
+      'molecular_analysis.molecular_abnormality_result':
+        'molecular_abnormality',
+      'tumor_assessments.tumor_state': 'tumor_site_state',
+      'tumor_assessments.tumor_site': 'tumor_site_state',
+      'stagings.stage_system': 'stage',
+      'stagings.stage': 'stage',
+      'minimal_residual_diseases.mrd_result_numeric': 'mrd_result',
+      'minimal_residual_diseases.mrd_result_unit': 'mrd_result',
+      'labs.lab_result_numeric': 'lab_result',
+      'labs.lab_result_unit': 'lab_result',
+    },
+    relations: [], // empty
+  };
```

with default to normal behavior I meant the behavior before this feature was added, so like if the feature is disabled, so it wouldn't render the calculator.

If we default to hardcoded values and those values change in the data dictionary it would crash the page, or show errors. While if we keep the feature disabled unless all the necessary parts are in place we can avoid that scenario.

---

Thank you, i have revised the code and added comments to help for the future. It can be tested by making `filterToRelation = null`